spec/Section 7 -- Response.md

```diff
@@ -23,6 +23,24 @@ with key `data`. The value of this entry is described in the "Data" section. If
 the operation failed before execution, due to a syntax error, missing
 information, or validation error, this entry must not be present.
 
+When the response of the GraphQL operation is an event stream, the first value
+will be the initial response. All subsequent values may contain `label` and 
+`path` entries. These two entries are used by clients to identify the the 
+`@defer` or `@stream` directive from the GraphQL operation that triggered this
+value to be returned by the event stream. The combination of these two entries
+must be unique across all values returned by the event stream. 
+
+If the response of the GraphQL operation is an event stream, each response map
+must contain an entry with key `hasNext`. The value of this entry is `true` for
+all but the last response in the stream. The value of this entry is `false` for 
+the last response of the stream. This entry is not required for GraphQL 
+operations that return a single response map. 
+
+The GraphQL server may determine there are no more values in the event stream 
+after a previous value with `hasNext`: `true` has been emitted. In this case 
```

Comment:
```suggestion
after a previous value with `hasNext: true` has been emitted. In this case 
```

Reply:
or

```suggestion
after a previous value with `hasNext` equal to `true` has been emitted. In this case 
```

Reply:
I wonder if we should use "entries" rather than "values" in the event stream, then we can use "after a previous entry with `hasNext` with value `true`"

Reply:
@benjie this section already uses the terms "entry"/"entries" to refer to fields on an object, e.g "If the operation encountered any errors, the response map must contain an entry with key `errors`" so I think it could be confusing to also use it for values from the event stream.